src/Hedgehog/Journal.fs

```diff
@@ -0,0 +1,24 @@
+namespace Hedgehog
+
+[<Struct>]
+type Journal =
+    | Journal of seq<unit -> string>
+
+module Journal =
+    let ofList (xs : seq<unit -> string>) : Journal =
+        Journal xs
+
+    let toList (Journal xs : Journal) : List<string> =
+        Seq.toList <| Seq.map (fun f -> f ()) xs
+
+    let empty : Journal =
+        Seq.empty |> ofList
+
+    let singleton (x : string) : Journal =
+        Seq.singleton (fun () -> x) |> ofList
+
+    let delayedSingleton (x : unit -> string) : Journal =
+        Seq.singleton x |> ofList
```

Comment:
Since `Journal` wraps many items of type `unit -> string`, so I think it makes more sense to give the preferred name of `singleton` to the function that accepts `unit -> string` and call the other function `eagerSingleton`.

Reply:
...or maybe `literalSingleton` or `constantSingleton`.  Something like that.

Reply:
Done